subprojects/core/src/main/java/org/gradle/api/internal/project/taskfactory/DefaultTaskClassInfoStore.java

```diff
@@ -125,10 +125,6 @@ public IncrementalTaskActionFactory(Class<? extends Task> taskType, Method metho
             this.method = method;
         }
 
-        public Method getMethod() {
```

Comment:
despite the doc telling they limit scope to private like all the other tools, it failing for this and others. Now making it kind of an (potential breaking) API change, as @elharo advised once.

As idea marks these elements as well it seems kind of save and interesting as errorprone must have higher scope awareness in comparison to PMD.